src/box2d.rs

```diff
@@ -215,6 +215,13 @@ where
 
     #[inline]
     pub fn union(&self, other: &Self) -> Self {
```

Comment:
Can you add a doc comment that describes the new behaviour?

Reply:
Done.